src.es5.js

```diff
@@ -4,6 +4,25 @@ var config = {
 	divider: '\n---'
 };
 
+function onLinkedInLoad() {
+	IN.Event.on(IN, "auth", getProfileData);
+}
+
+// Handle the successful return from the API call
+function onSuccess(data) {
```

Comment:
OK, you are right. I will be happy to improve my PR.
Can you give me more details on how I can implement this?